Swift/Chapter5/WorldTrotter/WorldTrotter/ConversionViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  ConversionViewController.swift
+//  WorldTrotter
+//
+//  Created by Arie Myrmo on 8/30/18.
+//  Copyright © 2018 Myrmo. All rights reserved.
+//
+
+import UIKit
+
+class ConversionViewController: UIViewController, UITextFieldDelegate {
+
+    // MARK: Properties and Outlets
+
+    @IBOutlet var celsiusLabel: UILabel!
+    @IBOutlet var textField: UITextField!
+    var fahrenheitValue: Measurement<UnitTemperature>? {
+        // Property observer - not triggered when the property value is changed from within an initializer
+        didSet {
```

Comment:
Adjust comment for the variable quick help

Reply:
Is it not obvious